lib/exercises.rb

```diff
@@ -1,21 +1,56 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
+# Space Complexity: O(n)
 
 def grouped_anagrams(strings)
```

Comment:
👍 Nice work, just a note if the strings are limited in size, the time complexity is O(n), otherwise it is O(n * m log m) where n is the number of strings and m is the length of the strings.